.travis.yml

```diff
@@ -49,6 +50,12 @@ matrix:
       sudo: required
       env: SWIFT_SNAPSHOT=swift-3.1-DEVELOPMENT-SNAPSHOT-2017-06-14-a
 
+before_install:
```

Comment:
I don't think this whole section (`before_install`) is needed. Can it be removed?